lib/osf-components/addon/components/search-result-card/file-secondary-metadata/template.hbs

```diff
@@ -16,7 +16,11 @@
                 @truncate={{5}}
                 as |list|
             >
-                <a href={{list.item.identifier}} target='_blank' rel='noopener noreferrer'>{{list.item.name}}</a>
+                {{#if list.item}}
+                    <a href={{list.item.identifier}} target='_blank' rel='noopener noreferrer'>{{list.item.name}}</a>
```

Comment:
I know this was done before, but out of curiosity, why are we not using OsfLinks here? That would handle the rel stuff automatically.

Reply:
I think this just didn't cross our minds when making the cards...

Reply:
Ah, well. Might be nice to fix at some point, but not now.